Gems/AWSClientAuth/Code/Source/AWSClientAuthSystemComponent.cpp

```diff
@@ -163,7 +164,11 @@ namespace AWSClientAuth
 
     void AWSClientAuthSystemComponent::OnSDKInitialized()
     {
-        Aws::Client::ClientConfiguration clientConfiguration;
+        AWSCore::AwsApiJobConfig* defaultConfig;
+        AWSCore::AWSCoreRequestBus::BroadcastResult(defaultConfig, &AWSCore::AWSCoreRequests::GetDefaultConfig);
+        Aws::Client::ClientConfiguration clientConfiguration =
```

Comment:
please add corresponding change to gamelift client ConfigureClient function if it is required for android platform.

Reply:
We can cut a separate ticket to track this.